test/test_other.py

```diff
@@ -14269,6 +14269,44 @@ def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
 
+  @only_windows('Check that directory permissions are properly retrieved on Windows')
+  @requires_node
+  def test_windows_nodefs_execution_permission(self):
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
```

Comment:
Is this needed?  Maybe just remove these two lines?

Reply:
As shared [here](https://github.com/emscripten-core/emscripten/pull/21902#discussion_r1593703949), probably not. I'll remove it.